src/services/DeepDaoTypes.ts

```diff
@@ -0,0 +1,28 @@
+export interface IDeepDaoGovernanceItem
+{
+  platform: string,
+  id: string,
+  name: string,
+  address: string,
+}
+
+export interface IDeepDaoOrganization
```

Comment:
![image](https://user-images.githubusercontent.com/30693990/185588210-e27c7a90-523a-4eef-bbd9-7373724a14cf.png)

Where does this interface come from?
They don't have docs around what the API returns?

jsdocs = comment you add to your code
```js
/*
 * like this
 */
export interface IDeepDaoOrganization
```